src/main.tsx

```diff
@@ -2,10 +2,17 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import router from "./router.tsx";
 import { RouterProvider } from "react-router-dom";
+import { Box } from "@mui/material";
 import "./index.css";
 
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
-    <RouterProvider router={router} />
+    <Box
+      sx={{
+        margin: -1,
+      }}
+    >
```

Comment:
this is just defalult html `<body/>` behavior, you could simply set `style={{margin:0}}` in `index.html` for not to overcomplicate this file